collect_app/src/main/java/org/odk/collect/android/tasks/DiskSyncTask.java

```diff
@@ -136,8 +136,11 @@ protected String doInBackground(Void... params) {
                                 uriToUpdate.add(new UriFile(updateUri, sqlFile));
                             }
                         } else {
+                            // remove it from the database also
+                            mFormsDao.deleteFormsForFormId(mCursor.getString(
```

Comment:
I think we can discuss whether we want to delete one by one like here? Or delete multiple by collecting id to be deleted here and do the deletion after we have all of the ids? Thoughts?